documentation/guides/generating-components.md

```diff
@@ -28,10 +28,12 @@ In response to the above, the component name should be the dashified version of
+The next place to look is in `node_modules/@spectrum-css/spectrumpattern/metadata/spectrumpattern.yml`. Here, you will find complete Spectrum CSS’s HTML representation of the many states, variants, and capabilities offered by the pattern that you are working with. The content of this file is also found on the <sp-link href="https://opensource.adobe.com/spectrum-css">Spectrum CSS documentation site</sp-link> under the name of the pattern you’re leveraging. Following this as a guide while you implement this component ensures that you will fulfill the expected features of this pattern.
 
-With this scaffold as a baseline in conjunction with these <sp-link href="guides/developing-components">instructions</sp-link> on working with the web components in general and the Spectrum Web Components repository in specific and the specification for the <sp-link href="guides/spectrum-config">`spectrum-config.js`</sp-link> outlining how to translate the Spectrum CSS source you will be working from into shadow DOM optimized CSS you are well on your way to adding a new package to the project. If you run into any issues with the instructions above of linked to across this documentation site, please feel free to raise <sp-link href="https://github.com/adobe/spectrum-web-components/issues">quesitons and issues</sp-link> on GitHub either as brand new issues when specifically related to the process of generating a component or as comments on the broader issue of adding a specific pattern when in reference to bringing a specific component to live.
+Now that you have this scaffold as a base, check out <sp-link href="guides/developing-components">these instructions</sp-link> on working with web components in general, and the <sp-link href="guides/spectrum-config">annotated `spectrum-config.js`</sp-link> file, which outlines how to translate the Spectrum CSS source into shadow DOM-optimized CSS. You are well on your way to adding a new component package to the project.
 
-Thanks for stopping by, we look forward to your contribution!
+If you run into any issues with the instructions above, or the ones linked across this documentation site, please feel free to raise questions or issues on <sp-link href="https://github.com/adobe/spectrum-web-components/issues">GitHub</sp-link>. You can do this either by creating a brand new issue when it specifically relates to the process of generating a component, or by leaving a comment on an already-created issue for a specific pattern when it concerns bringing that component to life.
```

Comment:
This is great. Should be enable the "Dicussions" feature for this? https://docs.github.com/en/discussions

Reply:
I think that would be a great thing for SWC!

Reply:
Feel free to do this in the follow up work, with the most recent `yarn new-package` workflow, but we should also include this here: https://github.com/adobe/spectrum-web-components/discussions for `raise questions`.